gammapy/background/tests/test_energy_offset_array.p

```diff
@@ -0,0 +1 @@
+y
```

Comment:
This looks like some file that you committed accidentally.
Remove it in a new commit.
For the future, always run `git status` and double-check what's about to be committed before running `git commit`.